i18n/fr/docusaurus-plugin-content-docs/current/unraid-os/release-notes/7.2.0.md

```
@@ -43,7 +43,7 @@ Vous pouvez maintenant étendre vos pools RAIDZ1/2/3 à un seul VDEV, un disque
 - Correctif : Un avertissement "expansion invalide" s'affichera désormais si le pool doit être mis à niveau d'abord
 - Amélioration : Meilleurs paramètres par défaut pour les vdevs ZFS RAIDZ
 
-#### Prise en charge des systèmes de fichiers Ext2/3/4, NTFS et exFAT {/* #ext234-ntfs-and-exfat-support */}
+#### Prise en charge des systèmes de fichiers Ext2/3/4, NTFS et exFAT {/* #ext234-ntfs-and-exfat-support */} {/* #ext234-ntfs-and-exfat-support */}
```

markdownlint-cli2 (0.22.1)

[warning] Spaces inside emphasis markers

(MD037, no-space-in-emphasis)